css/sass/_ticket.scss

```diff
@@ -30,6 +30,7 @@
 	td {
 		padding: 1.5rem;
 		border-top: 1px dashed #ddd;
+		background-color: #fff
```

Comment:
@sadi22 awesome work! One small thing pls, can we add the terminator (`;`) here?